packages/core/src/components/index.ts

```diff
@@ -36,9 +36,6 @@ export * from "./menu/menu";
 export * from "./menu/menuDivider";
 export * from "./menu/menuItem";
 export * from "./navbar/navbar";
-export * from "./navbar/navbarDivider";
-export * from "./navbar/navbarGroup";
-export * from "./navbar/navbarHeading";
```

Comment:
oh wait I think we should still export these, so you can use either `Navbar.Heading` _or_ `NavbarHeading`. though I'm open to being convinced otherwise.